### PR TITLE
Added recipe for pytest-pep8

### PR DESCRIPTION
**Pull request by bjodah**

### CODE REVIEW

File: recipes/pytest-pep8/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - pytest >=2.4.2
+    - pep8 >=1.3
+
+test:
+  imports:
+    - pytest_pep8
+
+about:
+  home: https://bitbucket.org/pytest-dev/pytest-pep8
+  license: MIT
```
Comment (jakirkham) [inline recipes/pytest-pep8/meta.yaml]: Please add...

```yaml
  license_file: LICENSE
```

**Comment by bjodah** [comment]
Strange... the appveyor build has actually completed...

File: recipes/pytest-pep8/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/pytest-pep8/meta.yaml]: Done.

**Comment by jakirkham** [comment]
No worries. Sometimes the webhook fails. No idea why, but it does happen sometimes. Normally is fixed with a restart, but am not interested in waiting around for the long queue to process it.

**Comment by jakirkham** [comment]
Thanks @bjodah.

**Comment by jakirkham** [comment]
Sorry I totally forgot to ask. 😳

Do you have any interest in helping maintain this, @nicoddemus?

**Comment by nicoddemus** [comment]
Hey @jakirkham thanks for asking, I can be listed as a maintainer no problem.

**Pull request merged**